baremetal.go

```diff
@@ -0,0 +1,114 @@
+// Package vagrantssh provides host connectivity in go for system/integration
+// testing in a multi host environment. It supports two testbed environments viz.
+// baremetal and vagrant
+/*
+Copyright 2014 Cisco Systems Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package vagrantssh
+
+import (
+	"fmt"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+var unexpectedSetupArg = func(exptdTypes string, rcvdArgs ...interface{}) error {
```

Comment:
hmm. not sure how I feel about that, especially because it's the only one that way. do you object to making it a regular func right now?